extensions/hibernate-validator/deployment/src/main/java/io/quarkus/hibernate/validator/deployment/MethodValidatedAnnotationsTransformer.java

```diff
@@ -101,9 +101,7 @@ private boolean isJaxrsMethod(MethodInfo method) {
         // check superclass, but only the direct one since we do not (yet) have the entire ClassInfo hierarchy here
         DotName superClass = clazz.superName();
         if (!superClass.equals(IndexingUtil.OBJECT)) {
-            if (isJaxrsMethod(signatureKey, superClass)) {
-                return true;
-            }
+            return isJaxrsMethod(signatureKey, superClass);
```

Comment:
Please keep it the way it was. I agree it leads to the same result but the logic is slightly different and I prefer we keep it that way.

Reply:
change reverted